content/300-guides/200-deployment/110-deployment-guides/001-deploying-to-aws-lambda.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: 'Deploy to AWS Lambda'
+metaTitle: 'Deploy to AWS Lambda'
+metaDescription: 'Learn how to deploy Node.js and TypeScript applications based on Prisma Client to AWS Lambda.'
+---
+
+<TopBlock>
+
+This guide explains how to integrate Prisma in your existing project using [AWS Lambda](https://aws.amazon.com/lambda/) with the [Serverless Framework](https://www.serverless.com/framework/). For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
```

Comment:
> how to integrate Prisma in your existing project using

"Integrate" & "deploy" are two separate guides focused on completely different separate subjects.

I think "Integrate" would be about adding Prisma to a specific stack (library, technology, runtime), etc. Deployment, conversely, would be about the changes you would make to make it ready. If we take this approach, our guides would be more "modular" in a way.

If we separate these two guides, we can cross-link between them.